### PR TITLE
Update permissions for backend checks for cloudfront bucket.

### PR DESCRIPTION
**Pull request by MancunianSam**
Update the roles for the download files and av lambdas to allow them
access to the new bucket.

Update the bucket policy to allow the lambda roles to access the files.

### CODE REVIEW

File: lambda/templates/av_lambda.json.tpl [hunk 1]
```diff
@@ -20,7 +20,9 @@
       ],
       "Resource" : [
         "arn:aws:s3:::tdr-upload-files-dirty-${environment}/*",
```
Comment (TomJKing) [inline lambda/templates/av_lambda.json.tpl:22]: Maybe order alphabetically?

File: lambda/templates/download_files_lambda.json.tpl [hunk 2]
```diff
@@ -20,7 +20,9 @@
       ],
       "Resource" : [
         "arn:aws:s3:::tdr-upload-files-dirty-${environment}/*",
```
Comment (TomJKing) [inline lambda/templates/download_files_lambda.json.tpl:22]: Maybe order alphabetically?

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of very minor comments.

File: lambda/templates/av_lambda.json.tpl (hunk 1 above)
Comment (MancunianSam) [reply lambda/templates/av_lambda.json.tpl:22]: I've changed them round. We should be able to remove the original permissions once we've switched things over to the new bucket.

File: lambda/templates/download_files_lambda.json.tpl (hunk 2 above)
Comment (MancunianSam) [reply lambda/templates/download_files_lambda.json.tpl]: I've fixed this one as well.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**